packages/@azure/keyvault/rollup.config.js

```diff
@@ -1,30 +1,36 @@
 import nodeResolve from "rollup-plugin-node-resolve";
+import builtins from "rollup-plugin-node-builtins"
 /**
  * @type {import('rollup').RollupFileOptions}
  */
+
+const version = require("./package.json").version;
+const banner = [
+  "/*!",
+  " * Copyright (c) Microsoft and contributors. All rights reserved.",
+  " * Licensed under the MIT License. See License.txt in the project root for",
+  " * license information.",
+  " * ",
+  ` * Azure KeyVault SDK for JavaScript - ${version}`,
+  " */"
+].join("\n");
+
 const config = {
-  input: './esm/lib/keyVaultClient.js',
+  input: './dist-esm/lib/index.js',
   external: ["@azure/ms-rest-js", "@azure/ms-rest-azure-js"],
   output: {
-    file: "./dist/keyvault.js",
+    file: "./dist/index.js",
     format: "umd",
     name: "Azure.Keyvault",
     sourcemap: true,
     globals: {
       "@azure/ms-rest-js": "msRest",
       "@azure/ms-rest-azure-js": "msRestAzure"
     },
-    banner: `/*
- * Copyright (c) Microsoft Corporation. All rights reserved.
- * Licensed under the MIT License. See License.txt in the project root for
- * license information.
- *
- * Code generated by Microsoft (R) AutoRest Code Generator.
- * Changes may cause incorrect behavior and will be lost if the code is
- * regenerated.
- */`
+    banner: banner
   },
   plugins: [
+    builtins(),
```

Comment:
I don't know which part of my change triggered some rollup warnings, and had to use `builtins()` to fix those.  @bterlson @ramya-rao-a please help review this.

```
./dist-esm/lib/index.js → ./dist/index.js...
preferring built-in module 'url' over local alternative at 'C:\github\jeremymeng\azure-sdk-for-js\node_modules\url\url.js', pass 'preferBuiltins: false' to disable this behavior or 'preferBuiltins: true' to disable this warning
preferring built-in module 'util' over local alternative at 'C:\github\jeremymeng\azure-sdk-for-js\node_modules\util\util.js', pass 'preferBuiltins: false' to disable this behavior or 'preferBuiltins: true' to disable this warning
(!) Missing shims for Node.js built-ins
Creating a browser bundle that depends on 'url' and 'util'. You might need to include https://www.npmjs.com/package/rollup-plugin-node-builtins
(!) Unresolved dependencies
https://rollupjs.org/guide/en#warning-treating-module-as-external-dependency
url (imported by dist-esm\lib\utils.js)
util (imported by dist-esm\lib\utils.js)
(!) Missing global variable names
Use output.globals to specify browser global variable names corresponding to external modules
url (guessing 'url')
util (guessing 'util')
created ./dist/index.js in 590ms
```

Reply:
builtins will cause unresolvable audit warnings, will it not?

Reply:
this plugin also brought three `npm audit` errors, even though its latest version is used.

Reply:
I tried `preferBuiltins: true` earlier and got some other warnings. Let me try again.

Reply:
you just need to mark them as "external" I think. Avoid the globals plugin at all costs.

Reply:
adding them to external list helped. though I still got one warning
```
./dist-esm/lib/index.js → ./dist/index.js...
(!) Missing shims for Node.js built-ins
Creating a browser bundle that depends on 'url' and 'util'. You might need to include https://www.npmjs.com/package/rollup-plugin-node-builtins
created ./dist/index.js in 377ms
```
I looked at servicebus code and the difference there is servicebus has a separate browser rollup config which doesn't have `util` as external.  I'll probably work on this in a later change.

Reply:
I think this is because you have them in the globals config field. If you remove them from there, this error should go away...

Reply:
I added them into globals because I am still getting two warnings after marking them as externals. I guess I need a browser config.
```
./dist-esm/lib/index.js → ./dist/index.js...
(!) Missing shims for Node.js built-ins
Creating a browser bundle that depends on 'url' and 'util'. You might need to include https://www.npmjs.com/package/rollup-plugin-node-builtins
(!) Missing global variable names
Use output.globals to specify browser global variable names corresponding to external modules
url (guessing 'url')
util (guessing 'util')
```

Reply:
After adding the rollup commonjs plugin it now builds fine.